dkist/dataset/tiled_dataset.py

```diff
@@ -195,7 +196,7 @@ def plot(self, slice_index, share_zscale=False, figure=None, swap_tile_limits: L
         slice_index : `int`, sequence of `int`s or `numpy.s_`
             Object representing a slice which will reduce each component dataset
             of the TiledDataset to a 2D image. This is passed to
-            ``TiledDataset.slice_tiles``
+            ``TiledDataset.slice_tiles``, if each tile is already 2D pass ``slice_index=...`.
```

Comment:
Why `...` instead of `None`?

Reply:
Because `None` in numpy adds a dummy dimension, where as `...` is "the rest of the dimensions".

Reply:
Ah, gotcha. Thanks